README.md

```diff
@@ -1,26 +1,55 @@
 # Brigade's ESLint configuration
 
+This package provides Brigade's highly opinionated ESLint config (`.eslintrc`) as an extensible shared config.
```

Comment:
Remove `"highly opinionated"` from here. Lint configs are always opinionated, so this is just redundant.